Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.9.5-slim-buster as base
+
+RUN pip install flask
+RUN pip install poetry
+
+COPY *.toml . 
+
+RUN poetry install
+
+COPY . .
+
+ENTRYPOINT poetry run flask run --host 0.0.0.0 --port 5000
+
+EXPOSE 5000
+
+# Configure for production
+FROM base as production
```

Comment:
You could also set some env vars in your Dockerfile. Like `ENV FLASK_ENV=production` here